sample/src/main/res/layout/activity_main.xml

```diff
@@ -17,6 +17,12 @@
         tools:context="com.stepstone.stepper.sample.MainActivity">
 
         <Button
+            android:id="@+id/noFrag"
```

Comment:
Could you move the button somewhere near the bottom, please? The idea is to have most common examples at the top and I think that most people still rely on Fragments